lib/cli/commands/tree.js

```diff
@@ -65,18 +88,39 @@ tree.handler = async function(argv) {
 				const lastIdx = dependencies.length - 1;
 				const newConnectorIndices = [...connectorIndices];
 				if (!lastChild) {
-					newConnectorIndices.push(indentation * indentWidth);
+					newConnectorIndices.push(level * indentWidth);
+				}
+
+				if (level >= requestedLevels) {
+					const msg = chalk.dim.italic(`Dependencies above level ${level} are hidden`);
```

Comment:
To me this sounds a bit confusing to understand, mainly because of `above`. If you see level as a number, it's correct, but in a tree I rather see the higher levels as "below" / "deeper" when speaking relative to the root.

Reply:
True! I changed it to `below`